cma/evolution_strategy.py

```diff
@@ -1801,11 +1802,18 @@ def eval_scaling_vector(in_):
                     )
             elif isinstance(self.opts['CMA_sampler'], type):  # type(...) is type, inspect.isclass(.)
                 try:
-                    self.sm = self.opts['CMA_sampler'](stds * np.ones(N))
+                    if self.opts['CMA_sampler_options'] is None:
+                        self.sm = self.opts['CMA_sampler'](stds * np.ones(N))
+                    else:
+                        self.sm = self.opts['CMA_sampler'](stds * np.ones(N), **self.opts['CMA_sampler_options'])
                 except:
                     try:
-                        self.sm = self.opts['CMA_sampler'](N)
+                        if self.opts['CMA_sampler_options'] is None:                        
+                            self.sm = self.opts['CMA_sampler'](N, **sampler_opts)
```

Comment:
the `sampler_opts` argument looks like a bug. In general, maybe 

```python
    self.sm = self.opts['CMA_sampler'](stds * np.ones(N), 
                                       {} if self.opts['CMA_sampler_options'] is None else
                                       self.opts['CMA_sampler_options'])
```
is a shorter version.